HISTORY.rst

```diff
@@ -21,6 +21,17 @@ History
   * ``HookMan.ensure_is_valid``
   * ``HookMan.get_status``
 
+
+- Generated files now sport a "do not modify" comment header.
+
+- Generation of the bindings code for ``HookCaller`` is skipped if ``specs.pyd_name`` is not defined.
```

Comment:
In the docs for `Specs.pyd_name`:

```
Base name of the shared library for the bindings for the HookCaller class. If None, no bindings will be
        generated.
```

So this will appear in the reference API. I think that's good enough, what do you think?

Reply:
Okay, I think that the API reference it's enough for now. 👍